src/uint.rs

```diff
@@ -10,11 +10,11 @@ mod parse;
 
 pub use self::convert::AsU256;
 use crate::I256;
-use core::num::ParseIntError;
+use core::{mem::MaybeUninit, num::ParseIntError};
 
 /// A 256-bit unsigned integer type.
 #[derive(Clone, Copy, Default, Eq, Hash, PartialEq)]
-#[repr(transparent)]
+#[repr(align(32))]
```

Comment:
Why the alignment change?

Reply:
Oops. I was experimenting and apparently forgot to revert.